### DISCUSSION TITLE
VPS Setup Log In Problem

### DISCUSSION THREAD

**Discussion by dfreder1** [magicbug/Cloudlog#1235] [Q&A] [answered]
I am trying to get cloudlog running at https://cloudlog.me.net/ on a digital ocean server running Ubuntu 18.04, PHP 7.2, MySQL. The instructions at https://github.com/magicbug/Cloudlog/wiki/Installation go perfectly well, but I can't log in at https://cloudlog.me.net/index.php/user/login. I get the html to appear, but can't log in.

I'm logging in with the user "mycallsign" and password created in step 4. 

I am able to get into the MySQL db by "sudo mysql -u mycallsign -p", but not from the web form. 

Must be a permissions problem somewhere.

Nothing in the apache2 or mysql error logs. I am using the www-data user in apache2. I've double-checked the permissions in Step 3.  

Any thoughts?

**Comment by AndreasK79** [comment, marked as answer]
Step 4 is for creating a user in MySQL. That is not the user login for Cloudlog. The first time, you need to log in with username m0abc and password demo. When you have logged in, you will then create your Cloudlog user. When that is done, delete m0abc.

**Reply by dfreder1** [reply]
Thanks. I updated the wiki.
